CHANGELOG.md

```diff
@@ -12,68 +13,66 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add package and datastream agent privileges in the package endpoint [#1109](https://github.com/elastic/package-registry/pull/1109)
+
 ### Deprecated
 
 ### Known Issues
 
-
 ## [v1.22.0](https://github.com/elastic/package-registry/compare/v1.21.0...v1.22.0)
 
 ### Breaking changes
 
 ### Bugfixes
 
-* Update Go runtime to 1.21.3. [#1102](https://github.com/elastic/package-registry/pull/1102)
-* Raise an error if the value of environment variables used to set parameters are not valid [#1103](https://github.com/elastic/package-registry/pull/1103)
+- Update Go runtime to 1.21.3. [#1102](https://github.com/elastic/package-registry/pull/1102)
+- Raise an error if the value of environment variables used to set parameters are not valid [#1103](https://github.com/elastic/package-registry/pull/1103)
```

Comment:
Umm, why these changes? Do we need to do them?

Reply:
probably not my IDE did some formating on save will revert that